wildfly-users-web/src/main/java/gallb/wildfly/users/web/AuthorMB.java

```diff
@@ -0,0 +1,211 @@
+package gallb.wildfly.users.web;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.enterprise.context.SessionScoped;
+import javax.faces.application.FacesMessage;
+import javax.faces.context.FacesContext;
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import org.jboss.logging.Logger;
+
+import gallb.wildfly.users.common.IAuthor;
+import gallb.wildfly.users.common.LibraryException;
+import model.Author;
+
+/**
+ * Author manager.
+ * 
+ * @author sipost
+ */
+@Named("authorBean")
+
+@SessionScoped
+public class AuthorMB implements Serializable {
+
+	/**
+	 * 
+	 */
+	private Logger oLogger = Logger.getLogger(AuthorMB.class);
+	private static final long serialVersionUID = -4702598250751689454L;
+
+	@Inject
+	private IAuthor oAuthorBean;
+
+	/**
+	 * 
+	 */
+	private List<Author> authorList = new ArrayList<>();// Currently displayed
+														// authors.
+	private Author currentAuthor = null;// Currently selected author.
+
+	/**
+	 * Sets faces context error message.
+	 * 
+	 * @param message
+	 */
+	public void error(String message) {

```

Comment:
Error, info, warn, fatal metodusok mar leteznek a ManagedBean osztalyban es majd mas MB-ben is fogjatok hasznalni, igy sok lesz a kod dupikalas.
Erdemes oket kiemelni egy masik osztalyba peldaul: MessageService.